### ISSUE TITLE
Flow Direction - is it reversed?

### ISSUE THREAD

**Issue by mickeyvip** [angular-ui/ui-layout#110]
Hello.

I noticed that the `flow` direction, which is stated to follow `flex-box` is actually behaves the opposite to the `flex-box`.

`flow: "column"` - render items side by side, although the **main axis** is set to `column`, meaning **vertical**, as it is in `flex-box`.
`flow: "row"` - render items one under another, although the **main axis** is set to `row`, meaning **horizontal** as it is in `flex-box`.

Here is an article from `CSS Tricks` with examples: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Or am I missing something?

Thank you.

**Comment by SomeKittens** [comment]
Good point.  Our "row" means "split things up into rows", rather than "arrange everything on the same row".  The docs could use some clearing up on that.

**Comment by BobbieBarker** [comment]
I'll take this one as well.
